src/js/angular/security/templates/user.html

```diff
@@ -141,8 +141,8 @@ <h3>{{'security.user.settings' | translate}}</h3>
 							{{'security.ignore.shared.queries' | translate}}
 						</label>
 
-                        <label class="d-block cookie-policy-link" ng-if="showCookiePolicyLink">
-                            <a href="#" ng-click="showCookiePolicy()">{{'cookie.cookie_policy_url_label' | translate}}</a>
+                        <label class="d-block" ng-if="showCookiePolicyLink">
+                            <button class="btn btn-secondary show-cookie-policy-btn" ng-click="showCookiePolicy()">{{'cookie.cookie_policy_url_label' | translate}}</button>
```

Comment:
The event handler for the click event here needs an event.preventDefault() becuase this button is inside a form which has a submit property and clicking the button leads to a redirect to the home page.